views/EditChart.react.js

```diff
@@ -13,22 +13,18 @@ const EditChart = ({ classes, graph, user }) => {
   const chartID = graph.chart_id
   const [chart, setChart] = useState()
   const handleSubmit = async (e, formValues) => {
-    try {
-      e.preventDefault()
-      const { data } = await editChart(chartID, formValues)
-
-      if (data.ok === 1) window.location.assign(routes.chart(chartID))
-    } catch (error) {
-      console.error(error)
-    }
+    e.preventDefault()
+    const { data } = await editChart(chartID, formValues)
+
+    if (data.ok === 1) window.location.assign(routes.chart(chartID))
   }
 
   useEffect(() => {
     getChart(chartID).then(({ data }) => setChart(data))
   }, [chartID])
 
   if (!chart) {
-    return null
+    return 'Chart not available to edit'
```

Comment:
do we need this? If the server redirects maybe we can keep this as it was?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/135/commits/839c57867329b3102e3cbacd8eb656a07915c622